src/rpc/mission.rs

```diff
@@ -44,11 +44,19 @@ impl MissionService for MissionRpc {
         &self,
         request: Request<mission::v0::StreamUnitsRequest>,
     ) -> Result<Response<Self::StreamUnitsStream>, Status> {
+        let request_message = request.into_inner();
```

Comment:
Nit: Might be my personal style, but I think I also see it a lot in other Rust projects. If you don't plan on using the old value of `request` any longer, you can simply re-use it:

```suggestion
        let request = request.into_inner();
```

Reply:
I didn't realize Rust would allow a new variable to overshadow an old one! Cool, I have adjusted it locally.